### ISSUE TITLE
折线图开启dataZoom后快速滑动出现数据错乱情况

### ISSUE THREAD

**Issue by LiuL0703** [apache/echarts#15254]
### Version
4.7.0

### Steps to reproduce
```
myChart.showLoading();

$.get(ROOT_PATH   '/data/asset/data/obama_budget_proposal_2012.list.json', function (obama_budget_2012) {
    myChart.hideLoading();

    option = {
        tooltip : {
            trigger: 'item'
        },
        legend: {
            data: ['Growth', 'Budget 2011', 'Budget 2012'],
            itemGap: 5
        },
        grid: {
            top: '12%',
            left: '1%',
            right: '10%',
            containLabel: true
        },
        xAxis: [
            {
                type : 'category',
                data : obama_budget_2012.names
            }
        ],
        yAxis: [
            {
                type : 'value',
                name : 'Budget (million USD)',
                axisLabel: {
                    formatter: function (a) {
                        a =  a;
                        return isFinite(a)
                            ? echarts.format.addCommas( a / 1000)
                            : '';
                    }
                }
            }
        ],
        dataZoom: [
            {
                type: 'slider',
                show: true,
                start: 94,
                end: 100,
                handleSize: 8
            },
            {
                type: 'inside',
                start: 94,
                end: 100
            },
            {
                type: 'slider',
                show: true,
                yAxisIndex: 0,
                filterMode: 'filter',
                width: 12,
                height: '70%',
                handleSize: 8,
                showDataShadow: false,
                left: '93%'
            }
        ],
        series : [
            {
                name: 'Budget 2011',
                type: 'line',
                data: obama_budget_2012.budget2011List
            },
            {
                name: 'Budget 2012',
                type: 'line',
                data: obama_budget_2012.budget2012List
            }
        ]
    };

    myChart.setOption(option);

});
```

### What is expected?
数据正常展示

### What is actually happening?
数据错乱
![image](https://user-images.githubusercontent.com/13818610/123619595-e913ef80-d83b-11eb-9fe7-40df3b89fee0.png)

**Comment by pissang** [comment]
@LiuL0703 可以暂时关闭更新动画

**Comment by pissang** [comment]
It should have been fixed in 5.2.1

**Comment by mideng2** [comment]
version 5.4.2 依旧有这个问题
